how-to/containerd-kata.md

```diff
@@ -185,14 +185,24 @@ The following configuration includes three runtime classes:
            SystemdCgroup = false
 ```
 
+From Containerd v1.2.4 and Kata v1.6.0, there is a new runtime option supported, by which you can specify a specific 
```

Comment:
Lines 188-189 suggested rewrite: 
From Containerd v1.2.4 and Kata v1.6.0, there is a new runtime option supported, which allows you to specify a specific kata configure file as follows: